### PR TITLE
test(e2e): add E2E tests for help-modal

### PR DESCRIPTION
**Pull request by sitek94**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Solves "template-challenge-components/help-modal" #51705

### CODE REVIEW

File: e2e/help-modal.spec.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    ).toBeVisible();
+    await expect(
+      page.getByRole('button', { name: translations.buttons.cancel })
+    ).toBeVisible();
+    await expect(
+      page.getByRole('button', { name: translations.buttons.close })
+    ).toBeVisible();
+  });
+
+  test('Create Post button closes the modal', async ({ page }) => {
```
Comment (sitek94) [inline e2e/help-modal.spec.ts:50]: I've been trying to test if it's correctly opening a new page with expected URL, something similar to example from the [Playwright docs](https://playwright.dev/docs/pages#handling-new-pages):

```ts
await page
  .getByRole('button', { name: translations.buttons['ask-for-help'] })
  .click();

const newPagePromise = context.waitForEvent('page');

page
  .getByRole('button', {
    name: translations.buttons['create-post']
  })
  .click()

const newPage = await newPagePromise;
await newPage.waitForLoadState();

await expect(newPage).toHaveURL(/.*\/forum\/.*/);
```

But I just couldn't get this test to pass. Will try tomorrow.

**Review by naomi-lgbt — APPROVED** [review]
Would be happy to land this as is. We can always expand on the tests in future PRs.

File: e2e/help-modal.spec.ts (hunk 1 above)
Comment (sitek94) [reply e2e/help-modal.spec.ts]: Alright, it seems that these tests were failing only locally. It works now:)

**Comment by sitek94** [comment]
@naomi-lgbt 

> Would be happy to land this as is. We can always expand on the tests in future PRs.

Actually, I gave it another shot and it worked... So it should be good to go:)

**Review by huyenltnguyen — APPROVED** [review]
LGTM. Thanks @sitek94!

**Pull request merged**